src/components/20-molecules/top-content-bar/story.js

```diff
@@ -0,0 +1,57 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+import './index';
+import { withMarkdown } from '../../../../.storybook/addons/markdown';
+import withBodyReset from '../../../../.storybook/addons/reset-body';
+import Readme from './README.md';
+
+storiesOf('Molecules/Top content bar', module)
+  .addDecorator(withMarkdown(Readme))
+  .addDecorator(withBodyReset())
+  .add(
+    'Top content bar - Default with button',
+    () =>
+      `<axa-top-content-bar ctatext="Click me">
```

Comment:
Shouldn't we use a real usecase in those stories?

Reply:
cant cause lit element doesnt camlise it